libs/elements/layout/carousel/src/lib/project-carousel/project-carousel.component.html

```diff
@@ -0,0 +1,22 @@
+    <!-- Slides -->
+<link rel="stylesheet" href="https://unicons.iconscout.com/release/v4.0.8/css/line.css"/>
+<section class="product">
+    <div class="container">
```

Comment:
Move the components inside a components folder. `lib/components/project-carousel`